docs/exchanges/binance-perpetual.md

```diff
@@ -8,35 +8,40 @@ tags:
 
 ## 📁 [Connector folder](https://github.com/hummingbot/hummingbot/tree/master/hummingbot/connector/exchange/binance)
 
+## 📝 Signup
+
+* Hummingbot referral link - <https://www.binance.com/en/futures/ref/hummingbot>
+* Sign up with Binance Futures using Hummingbot's referral link for a 10% discount!
```

Comment:
* Sign up with Binance Futures using Hummingbot's referral link for a 10% discount off trading fees